feature-libs/organization/account-summary/core/services/account-summary-page-meta.resolver.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from '@angular/core';
+import {
+  BreadcrumbMeta,
+  ContentPageMetaResolver,
+  Page,
+  RoutingService,
+  SemanticPathService,
+  TranslationService,
+} from '@spartacus/core';
+import { OrganizationPageMetaResolver } from '@spartacus/organization/administration/core';
+import { combineLatest, defer, Observable } from 'rxjs';
+import {
+  distinctUntilChanged,
+  map,
+  shareReplay,
+  switchMap,
+} from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AccountSummaryPageMetaResolver extends OrganizationPageMetaResolver {
+  constructor(
+    protected contentPageMetaResolver: ContentPageMetaResolver,
+    protected translation: TranslationService,
+    protected semanticPath: SemanticPathService,
+    protected routingService: RoutingService
+  ) {
+    super(contentPageMetaResolver, translation, semanticPath, routingService);
+  }
+
+  /**
+   * Breadcrumbs of the Account Summary page.
+   */
+  protected accountSummaryPageBreadcrumb$: Observable<BreadcrumbMeta[]> = defer(
```

Comment:
It is better to create const variables for string values, e.g. `'orgAccountSummary'`, so that customer can access them when extends this class.